pandas/core/indexes/datetimes.py

```diff
@@ -1904,24 +1904,47 @@ def delete(self, loc):
 
     def tz_convert(self, tz):
         """
-        Convert tz-aware DatetimeIndex from one time zone to another (using
-        pytz/dateutil)
+        Convert tz(timezone)-aware DatetimeIndex using pytz/dateutil.
+
+        When using DatetimeIndex providing with timezone this method
+        converts tz-aware DatetimeIndex from one timezone to another.
 
         Parameters
         ----------
         tz : string, pytz.timezone, dateutil.tz.tzfile or None
-            Time zone for time. Corresponding timestamps would be converted to
-            time zone of the TimeSeries.
+            Time zone for time.Corresponding timestamps would be converted
```

Comment:
You seemed to have removed a space between the two words, which I suppose was a mistake.